activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb

```diff
@@ -21,8 +21,9 @@ def query(sql, name = nil) # :nodoc:
           end
         end
 
+        COPY_TO = /copy\s+(.*)\s+to\s+(.*)/i
```

Comment:
How about using a local variable here? `COPY_TO` should not be constant because it should not be referenced outside.